<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>고급 설정</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/iconSmall.png</normaloff>icon/iconSmall.png</iconset>
  </property>
  <widget class="QGroupBox" name="advancedGroupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>고급 설정</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="bgProgramCloseButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>수동종료</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgProgramStartButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>수동시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="bgProgramStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>프로세스</string>
    </property>
   </widget>
   <widget class="QLabel" name="settingsFileStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>설정파일</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsFileCreateButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>수동생성</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingsFileDeleteButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>수동삭제</string>
    </property>
   </widget>
   <widget class="QLabel" name="startupLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>시작프로그램등록</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startupDisableButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>등록해제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startupEnableButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>등록</string>
    </property>
    <property name="ge=-ometry" stdset="0">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="startupLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>예약 취소 버튼</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="enableCancelRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>켜기</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="disableCancelRadioButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>끄기</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="startupLabel_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>101</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>종료하기</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="notifyTimeSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>65</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="startupLabel_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>101</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>초 전에 알리기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="notifyTimeApplyButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>적용</string>
    </property>
   </widget>
   <zorder>bgProgramStartButton</zorder>
   <zorder>bgProgramCloseButton</zorder>
   <zorder>bgProgramStatusLabel</zorder>
   <zorder>settingsFileStatusLabel</zorder>
   <zorder>settingsFileCreateButton</zorder>
   <zorder>settingsFileDeleteButton</zorder>
   <zorder>startupLabel</zorder>
   <zorder>startupDisableButton</zorder>
   <zorder>startupEnableButton</zorder>
   <zorder>startupLabel_2</zorder>
   <zorder>frame</zorder>
   <zorder>startupLabel_3</zorder>
   <zorder>notifyTimeSpinBox</zorder>
   <zorder>startupLabel_4</zorder>
   <zorder>notifyTimeApplyButton</zorder>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>고급 설정 활성화 비밀번호:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwTextBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="statusDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
